core/shared/src/main/scala/zio/prelude/NonEmptyTraversable.scala

```diff
@@ -168,6 +168,9 @@ trait NonEmptyTraversableSyntax {
       F.foreach1_(self)(f)
     def reduce(f: (A, A) => A)(implicit F: NonEmptyTraversable[F]): A                                             =
       F.reduce(self)(f)
+    // Another version to prevent name collision with the built-in `reduce`
```

Comment:
let's not be shy, a comment like this should be `/**`, not `//`, so that users in IDE with auto-completion can know right away what's going on.

Reply:
Gotcha! Upgraded to a bona fide doc string. And agreed re: waiting for other opinions :) Thanks, @sideeffffect!